IPV6_PKTINFO is the old name from RFC 2292 (section 4.4), where it has a
duel meaning.  Its *other* meaning is retained in RFC 3542 (section 4);
IPV6_RECVPKTINFO replaces this meaning there.  Winsock2 (and hence Cygwin
at the moment) only supports the RFC 2292 syntax.

--- origsrc/libtirpc-0.2.4/src/svc_dg.c	2013-12-09 14:59:51.000000000 -0600
+++ src/libtirpc-0.2.4/src/svc_dg.c	2014-06-09 17:08:43.131136100 -0500
@@ -653,6 +653,9 @@ svc_dg_enable_pktinfo(int fd, const stru
 		(void) setsockopt(fd, SOL_IP, IP_PKTINFO, &val, sizeof(val));
 		break;
 #ifdef INET6
+#ifndef IPV6_RECVPKTINFO
+#define IPV6_RECVPKTINFO IPV6_PKTINFO
+#endif
 	case AF_INET6:
 		(void) setsockopt(fd, SOL_IPV6, IPV6_RECVPKTINFO, &val, sizeof(val));
 		break;
